### ISSUE TITLE
"printedCards" or "officialCards" value request

### ISSUE THREAD

**Issue by daMoose52** [PokemonTCG/pokemon-tcg-api#61]
Request for a feature that provides the printed or official number of cards to a set in addition to the total number of cards in a set; IE cards are numbed as "1/82" where 82 is the number of officially announced cards in the set, and then secret cards are printed as "83/82", "84/82" etc. At the moment, "totalCards" reflects the number of entries in the set, so if there are 84 total cards, 82+2 secret rares, "totalCards" will report 84 cards to the set. I personally identify my cards as "Name X/Y", so that I can rapidly ID where they are from without being as familiar with some of the expansion symbols (or guessing on sets that are silly close) so I can punch in "Gardevoir 54/98", information gleaned off the card, and land on the XY Ancient Origins Gardevoir.

**Comment by daMoose52** [comment]
Attached, I've included a file as of Crimson Invasion; the totalCards column is from your DB, the other columns are some recommendations to help parse the sets out. Your totalCards appears off in a handful of places according to the numbers and breakdowns I found. 
This includes: officialCards (what I was looking for), subsetCards (for sets with the RC sub groups, the Arceus set, etc), and secretCards, for the secret rares. "printedCards" then is the total of all of those. 

[pkmn_card_counts.txt](https://github.com/PokemonTCG/pokemon-tcg-api/files/1687197/pkmn_card_counts.txt)
